eng/common/pipelines/templates/steps/verify-changelogs.yml

```diff
@@ -6,6 +6,12 @@ parameters:
   default: succeeded()
 
 steps:
+  - pwsh: |
+      Get-ChildItem -Recurse -Path "$(Build.SourcesDirectory)/sdk" -Include ci.yml `
+        | ForEach-Object { Get-Content $_ -Raw | yq -o=json > "$($_.FullName).json" }
+    displayName: Generate ci.yml.json
```

Comment:
Why write these to disk as opposed to just doing it in the verify-changelog script directly?

Reply:
Making the dependency a check in the package, and invoking there directly. Change coming.

Reply:
Ok this is done.